File: Python/1116.py
# BEE 1116 | DIVIDINDO X POR Y

N = int(input())

for i in range(N):
    line = input().split(' ')
    X = int(line[0])
    Y = int(line[1])
    if Y == 0:
        print("divisao impossivel")
    else:
        print(X / Y)

File: PHP/1116.php
<?php

# BEE 1116 | DIVIDINDO X POR Y

$N = (int) fgets(STDIN);

while($N--) {
    $nums = explode(" ", fgets(STDIN));
    $X = $nums[0];
    $Y = $nums[1];

    if ($Y == 0) echo "divisao impossivel" . PHP_EOL;
    else echo number_format($X/$Y, 1, '.', '') . PHP_EOL;

}